### PR TITLE
Allow numeric portions of strings, like "17.46 million"

### PR DESCRIPTION
**Pull request by hughescr**
Simple addition that allows strings that are partially numeric, partially words, like people commonly type.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -148,6 +148,13 @@ function convert(obj) {
 
     partial[cls] = 0;
 
+    // handle numeric values
+    if(vAmount.match(/^[0-9.]+$/))
```
Comment (svlapin) [inline index.js]: Please use Google code style https://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml?showone=Code_formatting#Code_formatting to be consisted with the rest of code:
- one space after `if`
- opening curly brace on the same line as `if`
- one space before opening curly brace

File: index.js [hunk 2]
```diff
@@ -148,6 +148,13 @@ function convert(obj) {
...
+    {
+        partial[cls] += parseFloat(vAmount);
```
Comment (svlapin) [inline index.js]: 2 space indent please

File: test/convert.js [hunk 3]
```diff
@@ -41,6 +41,15 @@ var data = [
       billions: 'one hundred and thirty five'
     },
     expect: 135000000000
+  },
+
+  {
+    input: {
+        billions: '17',
```
Comment (svlapin) [inline test/convert.js]: 2 space indent please

**Comment by svlapin** [comment]
@hughescr thanks a lot for your PR, I appreciate it! Please kindly fix code style issues to be consisted with the rest of code and I will merge it.

**Comment by hughescr** [comment]
There you go.  Fixed the style issues, I think.

**Comment by svlapin** [comment]
thanks @hughescr

**Pull request merged**